scripts/src/main/resources/scripts/migrations/2022.12.001

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+echo "test migration to 2022.12.001"
```

Comment:
Great to have a test, but we should create something real instead.
For the moment we could simply remove this having no migrations at all.
Once the feature is released (for a while), we could add a migration to address [tool excuses](https://github.com/devonfw/ide/issues/826#issuecomment-1271311469). So we should have `software/mvn` instead of `software/maven` as the commandlet is also named `mvn` and the same also for `node` instead of `nodejs`.